site/content/blog/fosdem-2020-ibex-faster/index.md

```diff
@@ -0,0 +1,20 @@
++++
+date = "2020-01-29T15:00:00+00:00"
+title = "At FOSDEM you can hear how we made our Ibex CPU core faster"
```

Comment:
Please add a `author = Greg Chadwick` here, and then you can remove the author at the bottom. (This way it will also appear in the feed/other metadata.)

Reply:
Thanks, done!